src/query/functions/binary/and_test.go

```diff
@@ -91,5 +92,6 @@ func TestAndWithSomeValues(t *testing.T) {
 	expected := values1
 	expected[0][1] = math.NaN()
 	expected[1][0] = math.NaN()
+	fmt.Println(expected, sink.Values)
```

Comment:
need to remove this?

Reply:
Oops yeah made a mental note to clean these but forgot it unfortunately